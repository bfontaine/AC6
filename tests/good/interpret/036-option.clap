def choose (x) =
  case { None => x | Some [y] => y }

val z = choose 42 (None)
val y = choose 0 (Some [42])

