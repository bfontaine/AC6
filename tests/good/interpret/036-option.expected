:- choose = <code>
:- z = 42
:- y = 42
